interface/modules/custom_modules/oe-module-claimrev-connect/public/debug-info.php

```diff
@@ -0,0 +1,69 @@
+<?php
+    namespace OpenEMR\Modules\ClaimRevConnector;
+    $tab="connectivity";
+    require_once "../../../../globals.php";
+
+    use OpenEMR\Modules\ClaimRevConnector\Bootstrap;
+    use OpenEMR\Modules\ClaimRevConnector\ClaimRevApi;
+    
+    class ConnectivityInfo
+    {
+        public $client_authority;
+        public $clientId;
+        public $client_scope;
+        public $client_secret;
+        public $api_server;
+        public $hasToken;
+        public $defaultAccount;
+        function __construct() {
+            $bootstrap = new Bootstrap($GLOBALS['kernel']->getEventDispatcher());
+            $globalsConfig = $bootstrap->getGlobalConfig();
+            $this->client_authority = $globalsConfig->getClientAuthority();     
+            $this->clientId = $globalsConfig->getClientId();
+            $this->client_scope = $globalsConfig->getClientScope();
+            $this->client_secret = $globalsConfig->getClientSecret();       
+            $this->api_server = $globalsConfig->getApiServer();     
+            $this->token = ClaimRevApi::GetAccessToken(); 
+            $this->hasToken = ClaimRevApi::CanConnectToClaimRev();
+            $this->defaultAccount = ClaimRevApi::getDefaultAccount($this->token); 
+        }
+    }
+
+?>
+<html>
+    <head>
+        <link rel="stylesheet" href="../../../../../public/assets/bootstrap/dist/css/bootstrap.min.css">
+    </head>
+    <title>ClaimRev Connect - Account</title>
+    <body>
+        <div class="row">
+            <div class="col">
+                <?php include '../templates/navbar.php'; ?>
```

Comment:
Generally use require in these situations so script will break if this file does not exist

Reply:
done